framework/src/executioners/FixedPointSolve.C

```diff
@@ -492,7 +492,8 @@ FixedPointSolve::computeCustomConvergencePostprocessor()
     _pp_scaling = *_fixed_point_custom_pp;
   _pp_new = *_fixed_point_custom_pp;
 
-  auto ppname = getParam<PostprocessorName>("custom_pp");
+  auto ppname = isParamValid("custom_pp") ? getParam<PostprocessorName>("custom_pp")
+                                          : getParam<PostprocessorName>("picard_custom_pp");
```

Comment:
I can't find this parameter name. Where is it?

Reply:
it s in PicardSolve

Reply:
If you use the relatively new `deprecateParam` API on the `PicardSolve::validParams`, then lookups of "custom_pp" will also consider if the user has set "picard_custom_pp" and then you don't have to check both here (which also has the advantage of not referencing a derived classes parameter name here in the base class)

Reply:
this doesnt work as well here because we are deprecating the parameter of the derived class with a parameter of the parent class. 

so if I did that I would allow the deprecated parameter in the other derived classes

Reply:
right, that's what I was suggesting

Reply:
let s see if it works